src/server.h

```diff
@@ -1596,6 +1596,7 @@ struct redisServer {
                                    xor of NOTIFY_... flags. */
     /* Cluster */
     int cluster_enabled;      /* Is cluster enabled? */
+    int cluster_port;   
```

Comment:
Since everything else in server.h is documented, probably should document this as well for consistency.